README.md

```diff
@@ -77,11 +77,17 @@ A note regarding "open a shell" in the following sections: this implies that the
 * On RHEL-based Linux, run e.g. `source /opt/rh/devtoolset-8/enable`.
 * On macOS, there is usually no action required (the current Xcode command line tools are activated by default).
 
+_Note: on Windows, replace `bin` with `Scripts` in the following commands. Some commands also differ in their file extension between Linux and Windows (`.sh` vs `.bat` and similar). Please refer to the `venv` documentation for details: https://docs.python.org/3.6/library/venv.html_
+
 #### Build Python Wheel
 
-1. Open a shell in the PyPRT git root.
-1. First time only: run `pipenv install` to get all required Python packages.
-1. Run `pipenv run python setup.py bdist_wheel`. This will build the CMake project and Python packages.
+1. Open a shell in the PyPRT git root. 
+1. First time only: setup a virtual Python environment with build dependencies for PyPRT.
+    1. Create the virtual environment: `python -m venv pyprt-venv`
```

Comment:
In https://docs.python.org/3/tutorial/venv.html, they suggest to start the name of the environment with a dot: ".pyprt-env" (to keep it hidden). This can be an idea. What do you think?

Reply:
good catch - done